volontulo/views/organizations.py

```diff
@@ -44,8 +45,24 @@ def get(request):
         )
 
     @staticmethod
+    @login_required
     def post(request):
         u"""Method responsible for saving new organization."""
+        if not (
+                request.POST.get('name') and
+                request.POST.get('address') and
+                request.POST.get('description')
+        ):
+            yield_message_error(
+                request,
+                u"Należy wypełnić wszystkie pola formularza."
+            )
+            return render(
+                request,
+                "organizations/organization_form.html",
+                {'organization': Organization()}
```

Comment:
I don't know, if that's good idea to send another new Organization() here.

For example, if You have filled description and forgot to fill address You will get an error and empty description field.